app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1186,14 +1193,39 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
             }
         }
 
+        private fun animateProgressBar(newProgress: Int, isLoading: Boolean) {
+            progressBarAnimation?.cancel()
+
+            if (pageLoadingIndicator.progress > newProgress) {
```

Comment:
Checks if the current progress is higher than the new one which means the animation would go from right to left rather than left to right making it a "deprogress toolbar" 
I could move it to a val but it reads fine to me, the alternative is something like `val isCurrentProgressHigherThanNewProgress` but we already know is doing that.

Reply:
ok, then based on you answer I would call it `shouldProgressDecrease`. As you said, we already see that we are checking which value is bigger but naming it using "why" we do that will help us.